modules/gitops-repo-server-properties.adoc

```diff
@@ -9,20 +9,17 @@
 The following properties are available for configuring the Repo server component:
 
 |===
-|*Name* |*Default* | *Description* 
-|`Resources` |`__<empty>__` |The container compute resources.
-|`MountSAToken` |`false` |Whether the `ServiceAccount` token should be mounted to the repo-server pod.
-|`ServiceAccount` |`""` |The name of the `ServiceAccount` to use with the repo-server pod.
-|`VerifyTLS` |`false` |Whether to enforce strict TLS checking on all components when communicating with repo server.
-|`AutoTLS` |`""` |Provider to use for setting up TLS the repo-server's gRPC TLS certificate (one of: openshift). Currently only available for OpenShift.
-|`Image` | `argoproj/argocd` |The container image for Argo CD Repo server. This overrides the `ARGOCD_REPOSERVER_IMAGE` environment variable.
-|`Version` | same as `.spec.Version` |The tag to use with the Argo CD Repo server.
-|`LogLevel` | `info` |The log level used by the Argo CD Repo server. Valid options are debug, info, error, and warn.
-|`LogFormat` | `text` |The log format to be used by the Argo CD Repo server. Valid options are text or json.
-|`ExecTimeout` | `180` |Execution timeout in seconds for rendering tools (e.g. Helm, Kustomize).
-|`Env` | `__<empty>__` |Environment to set for the repository server workloads.
-|`Replicas` | `__<empty>__` |The number of replicas for the Argo CD Repo server. Must be greater than or equal to `0`.
-|===
-
-
-
+|**Name** |**Default** | **Description**
+|`resources` |`__<empty>__` |The container compute resources.
+|`mountsatoken` |`false` |Whether the `serviceAccount` token should be mounted to the repo-server pod.
+|`ServiceAccount` |`""` |The name of the `serviceAccount` to use with the repo-server pod.
+|`verifytls` |`false` |Whether to enforce strict TLS checking on all components when communicating with repo server.
+|`autotls` |`""` |Provider to use for setting up TLS the repo-server's gRPC TLS certificate (one of: OpenShift). Currently only available for OpenShift.
```

Comment:
This can be rephrased similarly to what I suggested before:
```suggestion
|`autotls` |`""` |Provider to use for setting up TLS the repo-server's gRPC TLS certificate. Only the `openshift` value is currently available.
```

Reply:
Or maybe use "acceptable" instead of "available".